packages/dappmanager/package.json

```diff
@@ -65,6 +65,7 @@
     "express": "^4.17.1",
     "express-fileupload": "^1.1.5",
     "helmet": "^4.4.1",
+    "hs-client": "^0.0.10",
```

Comment:
Since this library just does a simple GET request could you implement this driver without any additional NPM package? Long term we want to minimize external dependencies :pray:

Reply:
Not sure if it makes sense as it is the official chain library and it is making 2 different API calls. The code to do node-fetch is much more messy.

```
    const nodeInfo = (await fetch(`${apiUrl}:${webPort}`, {
            headers: {
                'Authorization': 'Basic ' + Buffer.from(username + ":" + password).toString('base64')
            }
        }
    )).json();
```

Thoughts?

Reply:
This looks simple enough to me!